flow/Platform.cpp

```diff
@@ -2787,7 +2787,8 @@ void crashHandler(int sig) {
 	fprintf(stderr, "SIGNAL: %s (%d)\n", strsignal(sig), sig);
 	fprintf(stderr, "Trace: %s\n", backtrace.c_str());
 
-	_exit(128 + sig);
+	signal(sig, SIG_DFL);
+	kill(getpid(), sig);
```

Comment:
Seems like we shouldn't get an error from this or fail to die, but to be safe we could follow it up with a call to `_exit`.

Reply:
It seems if I call _exit right after kill, core doesn't actually get dumped /shrug

Reply:
Fair enough